rrfs-test/validated_yamls/templates/obtype_config/sfcshp_stationPressure_180.yaml

```diff
@@ -171,17 +168,17 @@
            test variables:
            - name:  ObsErrorData/stationPressure
            minvalue: 0.0
-           action:
-             name: reduce obs space
+           #action:
+           #  name: reduce obs space
 
          # Bounds Check (ObsValue)
          - filter: Bounds Check
            filter variables:
            - name: stationPressure
            minvalue: 20000.0
            maxvalue: 120000.0
-           action:
-             name: reduce obs space
+           #action:
+           #  name: reduce obs space
```

Comment:
Why do we comment out the `reduce obs space` here?

Reply:
same as above.